tests/HHClientLinterIgnoreTest.hack

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST;
+
+final class HHClientLinterIgnoreTest extends TestCase {
+  use HHClientLinterTestTrait;
+
+  public function getCleanExamples(): vec<(string)> {
+    return vec[
+      tuple("<?hh\nclass Foo {}"),
+      tuple(
+        '<?hh
+        function invalid_null_check(): void {
```

Comment:
- can you add a comment explaining which code should be raised?
- for both tests: add unclean examples
- for both tests (well, the trait): should test that there are lint errors in the 'clean' examples if a configuration is not provided